### PR TITLE
feature/COR-1913-extract-widetableprops

### PR DESCRIPTION
**Pull request by ben-van-eekelen**
## Summary
 
* Extract tableColumnWidths into seperate file

### CODE REVIEW

File: packages/app/src/components/tables/logic/column-widths.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+export const columnWidths: ColumnWidths = {
+  wide: {
+    defaultWidth: layout_302030,
+    behaviourTableTileWidth: layout_352030,
+    autumn2022ShotCoverageAgeGroupWidth: layout_102030,
+    primarySeriesCoveragePerAgeGroupWidth: layout_102030,
+  },
+  narrow: {},
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/components/tables/logic/column-widths.ts:30]: I think this one can be removed

**Review by VWSCoronaDashboard30 — APPROVED** [review]
Nicely done! Loved the column width setup!

File: packages/app/src/components/tables/logic/column-widths.ts (hunk 1 above)
Comment (ben-van-eekelen) [reply packages/app/src/components/tables/logic/column-widths.ts:30]: I left it in to indicate to someone picking this up later that `NarrowTable` can also make use of this definition file.

**Review by VWSCoronaDashboard30 — APPROVED** [review]
Solid work!

**Pull request merged**